packages/fxa-content-server/app/scripts/views/mixins/service-mixin.js

```diff
@@ -14,7 +14,7 @@ export default {
   },
 
   setInitialContext(context) {
-    context.set(this.relier.pick('service', 'serviceName'));
+    context.set(this.relier.pick('service', this.translate('serviceName')));
```

Comment:
Hi @LZoog, I'm poking around in this code to see whether I can fix https://github.com/mozilla/fxa/issues/1531, and I'm confused by the use of `this.translate` here. My understanding of `this.translate` is that this would try to find a localized version of the literal string "serviceName", then fall back to returning "serviceName" if no translation is found. I don't think we have a localization for this string, am I missing something?